<launch>
    <node pkg="major_project" name="navigation" type="navigation.py"/>
</launch>
